frontend/src/pages/lmEval/lmEvalResult/lmEvalResult.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+import { Link } from 'react-router';
+import { LMEvalContext } from '#~/pages/lmEval/global/LMEvalContext';
+import { downloadString } from '#~/utilities/string';
+import LMEvalResultApplicationPage from '#~/pages/lmEval/components/LMEvalResultApplicationPage';
+import LMEvalResultTable from './LMEvalResultTable';
+import { parseEvaluationResults } from './utils';
+
+const LMEvalResult: React.FC = () => {
+  const { evaluationName } = useParams<{ evaluationName: string }>();
+  const { lmEval } = React.useContext(LMEvalContext);
+
+  // Find the specific evaluation by name
+  const evaluation = React.useMemo(
+    () => lmEval.data.find((item) => item.metadata.name === evaluationName),
+    [lmEval.data, evaluationName],
+  );
```

Comment:
Fetching all the lmEval from context and filtering the required one will not be the optimal solution here. Instead of this just fetch the required lmEval via name and API